griptape/utils/__init__.py

```diff
@@ -23,6 +23,10 @@ def minify_json(value: str) -> str:
     return json.dumps(json.loads(value), separators=(",", ":"))
 
 
+def not_none(value):
+    return value is not None
+
+
```

Comment:
I don't think we should create a utility method for this.

Reply:
was hoping to sneak it past you